components/brave_wallet/browser/json_rpc_service.cc

```diff
@@ -1706,8 +1706,9 @@ void JsonRpcService::GetSolanaSignatureStatuses(
   auto internal_callback =
       base::BindOnce(&JsonRpcService::OnGetSolanaSignatureStatuses,
                      weak_ptr_factory_.GetWeakPtr(), std::move(callback));
-  return Request(solana::getSignatureStatuses(tx_signatures), true,
-                 mojom::CoinType::SOL, std::move(internal_callback));
+  return RequestInternal(solana::getSignatureStatuses(tx_signatures), true,
```

Comment:
Remove return to be consistent with other functions.

Reply:
done